### DISCUSSION TITLE
Can I alter the list of admin avatars for the dashboard?

### DISCUSSION THREAD

**Discussion by EmmiaSilk** [pocketbase/pocketbase#1701] [Q&A] [answered]
I would like to customize dashboard admin's avatar image for personal reasons. I've determined that the file I need to update is served as `/_/images/avatars/avatar0.svg`, but I haven't yet found a method of replacing it that I'm satisfied with. 

I tried adding it to `pb_public`, but paths that would serve in the dashboard are specially excluded from the static server, so this is not an option. Preferably, I would find a way to override files for the internal Svelte server, but this may not be an option.

If anyone has any insight into this, I'd love to hear.

**Comment by silenceofthewans** [comment, marked as answer]
I'm curious about your use case because those admin avatars are only for the admin panel. If you want the functionality where each admin can upload their own avatar instead of using the placeholders then this functionality hasn't been implemented yet. If you just want to change the images (avatar0.svg, avatar1.svg ...) then these are located in [ui folder](https://github.com/pocketbase/pocketbase/tree/master/ui/public/images/avatars) which is responsible for the svelte admin panel. This admin panel is "embeded" in the pocketbase's go binary. You would need to fork, change the images in `/ui/public/images/avatars` and then build.

Another option would be to use "Stylus" extension for adding custom CSS like this (_If it's only for visual purposes_):

```CSS
img[alt="Avatar 1"] {
    content: url("/api/files/...");
}
```

**Reply by EmmiaSilk** [reply]
I want to alter the admin avatars solely for self-indulgent personalization reasons. Since I'm the only person using the dashboard, styling with a browser extension is sufficient for my needs right now.
